source/uninstalling.rst

```diff
@@ -0,0 +1,70 @@
+.. _pdp.uninstalling:
+
+Uninstalling |pdp|
+********************************************************************************
+
+To uninstall |pdp|, remove all the installed packages and data / configuration files. 
+
+.. note::
+
+   Should you need the data files later, back up your data before uninstalling |pdp|.
+
+
+Using the |deb| format
+================================================================================
+  
+To uninstall |pdp| on platforms that use |apt| package manager such as Debian
+or Ubuntu, complete the following steps.
+
+Run all commands as root or via |sudo|.
+
+1. Stop the |pdp| service.
+
+   .. code-block:: bash
+
+      $ sudo systemctl start postgresql.service
+
+#. Remove the |percona-postgresql| packages
+
+   .. code-block:: bash
+
+      $ sudo apt-get remove percona-postgresql-12* percona-pgbackrest percona-patroni
+ 
+#. Remove configuration files         
+
+   .. code-block:: bash
+
+      $ rm -rf /etc/postgresql/12
+
+Using the |rpm| format
+================================================================================
+
+To uninstall |pdp| on platforms that use |yum| package manager such as 
+|rhel| or |centos|, complete the following steps.
+
+Run all commands as root or via |sudo|.
+
+1. Stop the |pdp| service.
+
+   .. code-block:: bash
+
+      $ sudo systemctl stop postgresql-12
+
+#. Remove the |percona-postgresql| packages
+
+   .. code-block:: bash
+ 
+      $ sudo yum remove percona-postgresql12*
+
+#. Remove configuration and data files
+
+   .. code-block:: bash
+
+      $ rm -rf /var/lib/pgsql
```

Comment:
Bad idea delete full pgsql directory because inside pgsql directory by default we have installed versions. As example:
`ls -la /var/lib/pgsql/12/data/
total 56
drwx------. 20 postgres postgres  4096 May 26 07:55 .
drwx------.  3 postgres postgres    36 May 26 07:56 ..
-rw-------.  1 postgres postgres     3 May 26 07:55 PG_VERSION
drwx------.  5 postgres postgres    41 May 26 07:55 base
drwx------.  2 postgres postgres  4096 May 26 07:55 global
drwx------.  2 postgres postgres     6 May 26 07:55 log
drwx------.  2 postgres postgres     6 May 26 07:55 pg_commit_ts
drwx------.  2 postgres postgres     6 May 26 07:55 pg_dynshmem
-rw-------.  1 postgres postgres  4269 May 26 07:55 pg_hba.conf
-rw-------.  1 postgres postgres  1636 May 26 07:55 pg_ident.conf
drwx------.  4 postgres postgres    68 May 26 07:55 pg_logical
drwx------.  4 postgres postgres    36 May 26 07:55 pg_multixact
drwx------.  2 postgres postgres    18 May 26 07:55 pg_notify
drwx------.  2 postgres postgres     6 May 26 07:55 pg_replslot
drwx------.  2 postgres postgres     6 May 26 07:55 pg_serial
drwx------.  2 postgres postgres     6 May 26 07:55 pg_snapshots
drwx------.  2 postgres postgres     6 May 26 07:55 pg_stat
drwx------.  2 postgres postgres     6 May 26 07:55 pg_stat_tmp
drwx------.  2 postgres postgres    18 May 26 07:55 pg_subtrans
drwx------.  2 postgres postgres     6 May 26 07:55 pg_tblspc
drwx------.  2 postgres postgres     6 May 26 07:55 pg_twophase
drwx------.  3 postgres postgres    60 May 26 07:55 pg_wal
drwx------.  2 postgres postgres    18 May 26 07:55 pg_xact
-rw-------.  1 postgres postgres    88 May 26 07:55 postgresql.auto.conf
-rw-------.  1 postgres postgres 26612 May 26 07:55 postgresql.conf
`
If we need to remove old data after update we need to delete only needed directory, but if we need full deletion it's ok.